corehq/apps/es/aggregations.py

```diff
@@ -480,9 +507,10 @@ class NestedTermAggregationsHelper(object):
     This is not at all related to the ES 'nested aggregation'.
     """
 
-    def __init__(self, base_query, terms):
+    def __init__(self, base_query, terms, aggregation=None):
```

Comment:
Example usage in the doc string would be good since it's not obvious how this class works. Also maybe rename `aggregation` to `bottom_level_aggregation`?